meinberlin/apps/votes/admin.py

```diff
@@ -14,6 +14,10 @@ class VotingTokenAdmin(admin.ModelAdmin):
     def module_name(self, token):
         return token.module.name
 
+    def save_model(self, request, obj, form, change):
```

Comment:
tokens created via the admin will get a new package number (i.e. the excel file will only contain one code), I don't see any other good way to do this. Appending it to an existing package is rather confusing? (especially when it has been downloaded already).